bundles/org.openhab.binding.ecobee/README.md

```diff
@@ -0,0 +1,1252 @@
+# Ecobee Binding
+
+[Ecobee Inc.](https://www.ecobee.com/)  of Toronto, Canada, sells a range of Wi-Fi 
+enabled thermostats, principally in the Americas. 
+This binding communicates with the 
+[Ecobee API](https://www.ecobee.com/home/developer/api/introduction/index.shtml) over a secure, 
+RESTful API to Ecobee's servers. 
+Monitoring ambient temperature and humidity, changing HVAC mode, changing heat or cool setpoints, 
+changing the backlight intensity, and even sending textual messages to one or a group of thermostats, 
+can be accomplished through this binding.
+
+## Supported Things
+
+The following thing types are supported:
+
+| Thing          |  ID             |  Description |
+|----------------|-----------------|--------------|
+| Account        | account         | Represents an Ecobee account. Manages all communication with the Ecobee API. |
+| Thermostat     | thermostat      | Represents a physical Ecobee thermostat associated with the Ecobee Account. |
+| Remote Sensor  | sensor          | Represents an Ecobee remote sensor that is associated with an Ecobee Thermostat. Also represents the internal sensor of an Ecobee Thermostat. |
+
+## Discovery
+
+Once an Account has been set up, and the API key has been authorized, Thermostats and Remote 
+Sensors will be discovered automatically.
+First, the thermostats will be added to the inbox.
+Then, once a Thermostat thing has been created, the Remote Sensors associated with that 
+thermostat will be added to the inbox.
+The binding will detect the capabilities (e.g. temperature, humidity, occupancy) supported by the 
+sensor, and then dynamically create channels for those capabilities.
+
+## Authorization
+
```

Comment:
This needs to be updated to indicate that the thing will be offline and will contain the pin that needs to be entered at ecobee.com. I'd think that this can replace the info about looking in the log file.

Reply:
Agreed. I reworked this entire section.